acceptance/localcluster/localcluster.go

```diff
@@ -564,9 +564,11 @@ func (l *Cluster) stop() {
 				log.Infof("node %d: stderr at %s", i, file)
 			}
 		} else if crashed {
+			// Don't remove crashed containers - could be useful for post-mortem.
```

Comment:
But that doesn't matter. If you're concerned about disk space - the volumes
aren't auto-deleted, so keeping the container around isn't really a space
issue.

On Fri, Oct 30, 2015 at 3:00 PM, Peter Mattis <notifications@github.com>
wrote:

> In acceptance/localcluster/localcluster.go
> <https://github.com/cockroachdb/cockroach/pull/2974#discussion_r43540246>:
>
> > @@ -564,9 +564,11 @@ func (l *Cluster) stop() {
> >  				log.Infof("node %d: stderr at %s", i, file)
> >  			}
> >  		} else if crashed {
> > +			// Don't remove crashed containers - could be useful for post-mortem.
>
> I just worry that if you default this behavior to true then we'll have
> various devs ending up with tons of old docker containers lying around. I
> don't think controlling this via an env variable is particularly onerous.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2974/files#r43540246>.
>